cmd/mimir/config-descriptor.json

```
@@ -2648,6 +2648,17 @@
           "fieldType": "map of tracker name (string) to matcher (string)",
           "fieldCategory": "advanced"
         },
+        {
+          "kind": "field",
+          "name": "out_of_order_time_window",
+          "required": false,
+          "desc": "Non-zero value enables out-of-order support for most recent samples that are within the time window w.r.t. the newest sample for that time series if it exists (i.e. within [series.maxTime-timeWindow, series.maxTime]) or w.r.t. the TSDB's max time if the series does not exist (i.e. within [db.maxTime-timeWindow, db.maxTime]). Ingester will need more memory that is a factor of rate of out of order sample being ingested and number of series getting out of order samples. It can be configured per-tenant.",
```

```suggestion
          "desc": "Non-zero value enables out-of-order support for the most recent samples that are within the time window in relation to the following conditions:
- The newest sample for that time series, if it exists. For example, within [series.maxTime-timeWindow, series.maxTime]).
- The TSDB's maximum time, if the series does not exist. For example, within [db.maxTime-timeWindow, db.maxTime]).
The ingester will need more memory as a factor of _rate of out-of-order samples being ingested_ and _the number of series that are getting out-of-order samples_.
You can configure it per tenant.",
```
This text is a bit of a beast. I am not sure how this will be formatted or if it will cause any problems.

---

This file is autogenerated :/ can't change it manually

---

I will apply the wording changes, not sure about adding newlines